.nvmrc

```diff
@@ -0,0 +1 @@
+18
```

Comment:
If you can go to 20 you should now as 20 was released in March

https://nodejs.dev/en/about/releases/

Reply:
Looks like it works :+1: